core/transformation/pathfinders/round_1/control_mappings.py

```diff
@@ -104,3 +108,43 @@ def _intervention_theme_to_standard_outcomes(standard_outcomes_df: pd.DataFrame)
         for _, row in standard_outcomes_df.iterrows()
         if not pd.isna(row["Intervention theme"])
     }
+
+
+def _standard_output_uoms(standard_outputs_df: pd.DataFrame) -> dict[str, str]:
+    """Creates a mapping from standard output to its unit of measurement."""
+    return {
+        row["Standard output"]: standard_outputs_df.loc[
+            standard_outputs_df["Standard output"] == row["Standard output"], "UoM"
+        ].tolist()
+        for _, row in standard_outputs_df.iterrows()
+        if not pd.isna(row["UoM"])
+    }
+
+
+def _standard_outcome_uoms(standard_outcomes_df: pd.DataFrame) -> dict[str, str]:
+    """Creates a mapping from standard outcome to its unit of measurement."""
+    return {
+        row["Standard outcome"]: standard_outcomes_df.loc[
+            standard_outcomes_df["Standard outcome"] == row["Standard outcome"], "UoM"
+        ].tolist()
+        for _, row in standard_outcomes_df.iterrows()
+        if not pd.isna(row["UoM"])
+    }
+
+
+def _bespoke_output_uoms(bespoke_outputs_df: pd.DataFrame) -> dict[str, str]:
+    """Creates a mapping from bespoke output to its unit of measurement."""
+    return {
+        row["Output"]: bespoke_outputs_df.loc[bespoke_outputs_df["Output"] == row["Output"], "UoM"].tolist()
+        for _, row in bespoke_outputs_df.iterrows()
+        if not pd.isna(row["UoM"])
+    }
+
+
+def _bespoke_outcome_uoms(bespoke_outcomes_df: pd.DataFrame) -> dict[str, str]:
+    """Creates a mapping from bespoke outcome to its unit of measurement."""
+    return {
+        row["Outcome"]: bespoke_outcomes_df.loc[bespoke_outcomes_df["Outcome"] == row["Outcome"], "UoM"].tolist()
+        for _, row in bespoke_outcomes_df.iterrows()
+        if not pd.isna(row["UoM"])
+    }
```

Comment:
Could we not just combine these and similar into one function calling params on the name of the columns?

Or do you reckon it's better to be verbose like this?